docs/docs/workflows/update-manifest.md

```diff
@@ -1,16 +1,15 @@
 # Updating a manifest
-To ensure the confidentiality of a deployed application, MarbleRun uses a manifest to define the software packages and the infrastructure your deployment uses. To verify that your deployment hasn't been altered, the manifest is usually set in stone after it was set to ensure no one can alter with your cluster.
 
-Yet, updates play an important role to ensure your software stays secure. To avoid having to redeploy your application from scratch, MarbleRun allows uploading a separate "update manifest" which increases the minimum `SecurityVersion` of one or multiple already deployed packages. After such an update is performed, an old version of a defined software package can't be loaded anymore under the current manifest.
+When [defining the initial manifest](define-manifest.md), you have to choose whether it should be [partially or fully updatable](../features/manifest.md) by assigning appropriate [roles](define-manifest.md#roles) to eligible users.
 
-## Requirements
-To deploy an update, you need the certificate/private key pair belonging to a user from the `Users` section of the original manifest, as described in ["defining a manifest."](../workflows/define-manifest.md#marbles)
-Furthermore the user needs to be [permitted to update](../workflows/define-manifest.md#roles) the chosen packages.
+## Package updates
 
-If no user with the permission for updates has been initially set up, no updates can be applied.
+Updates play an important role to ensure your software stays secure. To avoid having to redeploy your application from scratch, MarbleRun allows uploading a separate "update manifest" that increases the minimum `SecurityVersion` of already deployed packages. After such an update is performed, an old version of a defined software package can't be loaded anymore under the current manifest.
```

Comment:
```suggestion
Updates play an important role in ensuring your software stays secure. To avoid redeploying your application from scratch, MarbleRun allows uploading a separate "update manifest" that increases the minimum `SecurityVersion` of already deployed packages. After such an update is performed, an old version of a defined software package can't be loaded anymore under the current manifest.
```